### ISSUE TITLE
Schema tidy-up tasks

### ISSUE THREAD

**Issue by duncandewhurst** [GFDRR/rdl-standard#129]
These changes should already be mostly done, but in order to ensure that nothing was missed, once all the schema changes are merged into `dev`, we should:

- [ ] Replace any remaining `common_` codelist definitions in `$defs` with CSV codelists and context-specific field titles and descriptions
- [ ] Remove any unreferenced definitions from `$defs`
- [ ] Consider whether any definitions in `$defs` should be in-lined (definitions that are referenced only once, do not appear in arrays array, and are not expected to be reused)
- [ ] Consider reordering the definitions in `$defs` in-line with their use (e.g. grouping by hazard, exposure, vulnerability and loss, higher-level before lower-level subschemas)
- [ ] Add validation keywords by running OCDS Kit's [`schema-strict`](https://ocdskit.readthedocs.io/en/latest/cli/schema.html#schema-strict)
- [ ] Validate codelist enums (https://github.com/GFDRR/rdl-standard/pull/127)
- [ ] Check that all fields have titles and descriptions (https://github.com/GFDRR/rdl-standard/pull/128)

**Comment by duncandewhurst** [comment]
Moving straight to agreed and ready as we should be adding these keywords as we go anyway. Happy to discuss if needed, though.

**Comment by duncandewhurst** [comment]
Rather than creating many individual issues, I've updated the issue title and description to cover all the schema tidy-up tasks we'll need to do.
